### PR TITLE
fix for actions and clicommon etc

### PR DESCRIPTION
**Pull request by qiaozha**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ See documentation [here](doc/00-overview.md)
 
 ``` yaml
 use-extension:
-  "@autorest/clicommon": "latest"
+  "@autorest/clicommon": "/Users/zhangqiaoqiao/work/code/autorest.cli.common"
```
Comment (zikalino) [inline README.md]: this shouldn't be here

File: src/plugins/azgenerator/TemplateAzureCliParams.ts [hunk 2]
```diff
@@ -60,7 +60,7 @@ export function GenerateAzureCliParams(model: CodeModelAz): string[] {
     output.push(")");
 
     if (hasActions) {
-        output.push("from azext_" + model.Extension_NameUnderscored + ".action import (")
+        output.push("from azext_" + model.Extension_NameUnderscored + ".actions import (")
```
Comment (zikalino) [inline src/plugins/azgenerator/TemplateAzureCliParams.ts]: fixed in different way in another pr. correct filename is "action.py"

File: README.md (hunk 1 above)
Comment (qiaozha) [reply README.md]: the latest clicommon that contains the directive feature Rodge is working on hasn't been released yet. Please ignore this for now.

File: src/plugins/azgenerator/TemplateAzureCliParams.ts (hunk 2 above)
Comment (qiaozha) [reply src/plugins/azgenerator/TemplateAzureCliParams.ts]: but  in generator.py it still using actions.py

File: README.md (hunk 1 above)
Comment (RodgeFu) [reply README.md]: should have been fixed along with clicommon side

File: src/plugins/azgenerator/TemplateAzureCliParams.ts (hunk 2 above)
Comment (RodgeFu) [reply src/plugins/azgenerator/TemplateAzureCliParams.ts]: "action.py" is used now. Zim, please help to pull and merge if needed. thanks

**Pull request merged**